src/nvim/testdir/test_profile.vim

```diff
@@ -485,11 +485,11 @@ func Test_profdel_func()
   " - Foo1() is called twice (profdel not invoked)
   " - Foo2() is called once (profdel invoked after it was called)
   " - Foo3() is not called (profdel invoked before it was called)
-  call assert_equal('FUNCTION  Foo1()',               lines[0])
-  call assert_match('Defined:.*Xprofile_file.vim',    lines[1])
-  call assert_equal('Called 2 times',                 lines[2])
-  call assert_equal('FUNCTION  Foo2()',               lines[8])
-  call assert_equal('Called 1 time',                  lines[10])
+  call assert_equal('FUNCTION  Foo2()',               lines[0])
+  call assert_equal('Called 1 time',                  lines[2])
+  call assert_equal('FUNCTION  Foo1()',               lines[8])
+  call assert_match('Defined:.*Xprofile_file.vim',    lines[9])
+  call assert_equal('Called 2 times',                 lines[10])
```

Comment:
Why does this need change?

Reply:
Hmm, it seems removing `runtime/plugin/man.vim` changes this. This is strange.

Reply:
Yes it is. Very.